src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:regular,500"
+      rel="stylesheet"
+      >
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a class="logo"href="#homepage"><img src="images/logo.png"  alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item">
+            <a href="#apple" class="is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#samsung">
+              Samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#smartphones">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_item"data-qa="hover">
+            <a href="#laptops&computers">
+              Laptops & computers
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav_item">
            <a href="#laptops&computers" data-qa="hover">
              Laptops & computers
            </a>
          </li>
```

![image](https://user-images.githubusercontent.com/86923089/216814223-b0e6aacd-f980-41b2-b92c-830071254867.png)